lib/Core/TxTree.h

```diff
@@ -572,7 +633,8 @@ class TxTreeNode {
 /// from
 /// KLEE's Executor class, which is the core symbolic executor of KLEE.
 /// The main functionality of the TxTree#execute versions
-/// themselves is to simply delegate the call to TxDependency#execute and related
+/// themselves is to simply delegate the call to TxDependency#execute and
```

Comment:
fix this please.

Reply:
what to fix for this @rasoolmaghareh ?

Reply:
The word related has been moved to the next line and we have a one word comment in one line.